lib/src/hitmap.dart

```diff
@@ -173,6 +205,32 @@ Future<Map<String, Map<int, int>>> parseCoverage(
   return globalHitmap;
 }
 
+/// Returns a JSON hit map backward-compatible with pre-1.16.0 SDKs.
```

Comment:
Should we even support that low of an SDK? I'd rather just remove this logic and bump the min SDK.

Reply:
I'm fine with bumping the min SDK, but I don't know what logic in here is specific to older SDKs, so I'm not sure what changes you're asking for. Maybe we could do that in a follow up CL.

Reply:
I'm not sure either. That's a good question for @cbracken